lib/Conversion/LowerDAP/LowerDAPPass.cpp

```diff
@@ -84,14 +84,14 @@ class DAPBiquadLowering : public OpRewritePattern<dap::BiquadOp> {
     Value c0 = rewriter.create<ConstantIndexOp>(loc, 0);
     Value c1 = rewriter.create<ConstantIndexOp>(loc, 1);
     Value c2 = rewriter.create<ConstantIndexOp>(loc, 2);
-    Value c3 = rewriter.create<ConstantIndexOp>(loc, 3);
+    // Value c3 = rewriter.create<ConstantIndexOp>(loc, 3);
```

Comment:
If such variables are not used, can you not delete them? Is there any motive related to readability behind commenting them?

Reply:
In the kernel, `a0` is not being used in the algorithm. Therefore I thought to maintain uniform readability, its better to comment the variable to indicate that it is being not used. What's your opinion on this, is it recommended to delete them?

Reply:
Delete the variable declaration, instead add a comment for explanation.